src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -499,11 +499,6 @@ public async Task<int> DeployAsync()
                         Task compute = null;
                         if (configuration.UseAks)
                         {
-                            ConsoleEx.WriteLine();
```

Comment:
Why was this removed? I could understand moving it, but removing it? This is information needed for upgrading, similar to the other block for VM

Reply:
Discussed offline; will add more upgrade instructions